tmt/result.py

```diff
@@ -141,6 +142,7 @@ def from_test(
 
         _result = Result(
             name=test.name,
+            serial_number=test.serial_number,
```

Comment:
@happz will `results.yaml` also change?

Reply:
Yes, definitely. But, I have a problem with how `results.yaml` and `tests.yaml` are documented, and I plan to do something about it, including revamping the docs and adding JSON schemas for both. So, `results.yaml` does change, and `serial_number` starts appearing in results, but it's undocumented like some other fields are. If you won't mind, I'd leave this to a follow-up patch addressing documentation of both of these files in a more generous way.

Reply:
makes sense, ty!